install_builddeps.sh

```diff
@@ -50,7 +50,7 @@ install_freediameter() {
 
 install_grpc() {
 	$SUDO rm -rf /tmp/grpc
-	git clone -b $(curl -L https://grpc.io/release) \
+	git clone -b v1.26.0 \
```

Comment:
Lets understand from experts if this is right way of doing it. @pingping-lin @hyunsun  is this correct way ?